pkg/tracing/tracing.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2017 Google Inc.
```

Comment:
Got as close as is reasonable: the only branches not covered are error cases in the tracer impl and in gRPC's metadata class not maintaining its own invariants. To do more would require implementing fake tracer and metadata classes, which I don't think is worth the effort.